Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: test format lint
+
+test:
+	pytest --exitfirst
+
+format:
+	pip install -qqq --upgrade black isort && \
```

Comment:
Let us not install or upgrade any instances here. It's better to install all dependencies via something like `poetry install`, but let's discuss it later